doc/audit/index.rst

```diff
@@ -24,15 +24,21 @@ Audit
 
 
     :doc:`python_library`
-    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+    ^^^^^^^^^^^^^^^^^^^^^
 
     Utility library that can be used to write ledger audit scripts quickly and easily.
 
     ---
 
+    :fa:`shield-alt` :doc:`read_write_restrictions`
+    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+    Governance maps should only be written to when applying approved proposals.
```

Comment:
Except for the proposals and proposals_info maps, which are written to when proposals are submitted or withdrawn, or ballots submitted.

Reply:
Oh and also history and cose_history, which are written to in the same circumstances.

Reply:
This summary is brief and imprecise to fit on a single tile, but I've reworded slightly to be less wrong.